src/Main.cpp

```diff
@@ -617,6 +617,7 @@ class Plugin {
                 amx_PushString(script.amx, &addr_params, nullptr, params, 0, 0);
                 amx_Push(script.amx, playerid);
                 ExecAmxPublic(script.amx, &retval, iter_cmd->second.addr);
+                if(retval == 0) retval = -1;
```

Comment:
This doesn't follow the syntax flow of the rest of the script...

`
                if(retval == 0) {
                        retval = -1;
                }
`